builtin/game/privileges.lua

```diff
@@ -58,4 +58,7 @@ core.register_privilege("zoom", {
 	description = "Can zoom the camera",
 	give_to_singleplayer = false,
 })
-
+core.register_privilege("debug", {
+	description = "Can enable various dangerous debug options that may affect gameplay",
```

Comment:
'Dangerous' seems wrong here, how about:
"Allows enabling of various debug options that may affect gameplay"
Is this any good?